rules/lfi-os-files.data

```diff
@@ -48,118 +59,167 @@
 .psql_history
 .python_history
 .rediscli_history
-.Rhistory
+.rhistory
 .rhosts
 .sh_history
 .sqlite_history
-.ssh/authorized_keys
-.ssh/config
-.ssh/id_dsa
-.ssh/id_dsa.pub
-.ssh/id_rsa
-.ssh/id_rsa.pub
-.ssh/identity
-.ssh/identity.pub
-.ssh/id_ecdsa
-.ssh/id_ecdsa.pub
-.ssh/known_hosts
-.subversion/auth
-.subversion/config
-.subversion/servers
-.tconn/tconn.conf
+.ssh/
+.subversion/
+.tconn/
+.tor/
 .tcshrc
-.vidalia/vidalia.conf
+.vidalia/
+.vim/
 .viminfo
 .vimrc
 .www_acl
 .wwwacl
 .xauthority
 .zhistory
-.zshrc
 .zsh_history
-.nsconfig
+.zshrc
+apache/access.conf
+apache/apache.conf
+apache/apache2.conf
+apache/audit_log
+apache/conf
```

Comment:
There are a couple of duplicates with different prefixes and suffixes.

Reply:
I think I did remove them....